composer.json

```diff
@@ -106,7 +106,7 @@
         "php-http/client-common":         "^1.3",
         "php-http/message-factory":       "^1.0",
         "php-http/discovery":             "^1.0",
-        "ext-json": "^1.5"
+        "ext-json": "^1.5 || ^7.4"
```

Comment:
Maybe we can relax that value to `"ext-json": "*"`

Reply:
This is what I usually do as well.

Reply:
I was wondering the same... I'm fine with that. Let me change it.

Reply:
Done